guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input – such as a mouse, stylus, or finger on a touch screen – that only targets a single point on the screen (as opposed to multi-pointer/multi-touch scenarios)</p>
```

Comment:
```suggestion
   <p>an input that only targets a single point on the screen. Such as a mouse, stylus, or finger on a touch screen, in contrast to multi-pointer/multi-touch scenarios.</p>
```

Reply:
patrick suggested "on page" instead of "on screen"